### PR TITLE
OXT-1418: Updates for L1TF mitigations

### PR DESCRIPTION
**Pull request by jandryuk**
Uprev Linux to 4.14.66.  This required a fix to the OXT-1303 lookup_swap_cache bandaid (which I renamed).

Also uprev the microcode with commits from meta-intel.  This bring in support for SSBD and L1D_FLUSH.

This superseeds https://github.com/OpenXT/xenclient-oe/pull/1015

### CODE REVIEW

File: recipes-kernel/linux/4.14/defconfigs/openxt-installer/defconfig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.14.57 Kernel Configuration
+# Linux/x86 4.14.63 Kernel Configuration
```
[the remark this answers is not in the document]
Comment (jandryuk) [reply recipes-kernel/linux/4.14/defconfigs/openxt-installer/defconfig]: Yeah, I didn't update the defconfigs.  Thanks for checking.  Do you want me to add that in?

**Comment by jandryuk** [comment]
Have you generated the 'Bad swp_entry' messages?  I have seen those, but that is the bandaid working to avoid an Oops.

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1702

File: recipes-kernel/linux/4.14/defconfigs/openxt-installer/defconfig (hunk 1 above)
[the remark this answers is not in the document]
Comment (eric-ch) [reply recipes-kernel/linux/4.14/defconfigs/openxt-installer/defconfig]: Nope, no worries. I just needed a reminder to not check again today :).

**Comment by eric-ch** [comment]
@jandryuk Yes, plenty.

```
root@xenclient-dom0:~# dmesg | grep '\(Bad swp_entry\|bad pte\)'
[ 3042.451536] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback b7c00000 [b7c00000-b7c21000] file:  (null)
[ 3042.451551] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[ 3187.796992] Bad swp_entry: 207fffff 10 1ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 3187.797006] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000300000000)
[ 3260.347642] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 3260.347657] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 3437.825909] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 3437.825924] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[ 3733.355266] Bad swp_entry: 207fffff 10 1ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 3733.355296] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000300000000)
[ 3879.412085] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 3879.412100] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 4314.134055] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 4314.134071] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 5983.533102] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 5983.533118] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 6200.856695] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 6200.856710] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 8388.161837] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 8388.161853] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[ 9553.253447] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[ 9553.253461] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[10423.238500] Bad swp_entry: 207fffff 10 1ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[10423.238516] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000300000000)
[10858.223583] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[10858.223598] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[10931.609800] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[10931.609828] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[11291.549027] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[11291.549054] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[11866.948684] Bad swp_entry: 207fffff 10 1ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[11866.948710] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000300000000)
[13606.312547] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[13606.312563] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[13969.948760] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback 98fd000 [98fd000-991e000] file:  (null)
[13969.948775] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[14114.646369] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[14114.646397] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[14546.186629] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback 98fd000 [98fd000-991e000] file:  (null)
[14546.186645] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[15492.725781] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[15492.725807] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[15709.395366] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[15709.395381] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[16004.755351] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback 98fd000 [98fd000-991e000] file:  (null)
[16004.755379] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[16150.615257] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback 98fd000 [98fd000-991e000] file:  (null)
[16150.615284] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[16798.439253] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[16798.439266] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[16872.543344] Bad swp_entry: 21ffffff 10 7ff/0 974 tapback b7c01000 [b7c00000-b7c21000] file:  (null)
[16872.543359] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5183f1c(8000000400000000)
[17095.724957] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[17095.724986] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
[17312.026805] Bad swp_entry: 21ffffff 10 7ff/0 976 tapback 98fd000 [98fd000-991e000] file:  (null)
[17312.026820] /home/build/openxt/master/tmp-glibc/work-shared/xenclient-dom0/kernel-source/mm/swap_state.c:704: bad pte c5163f1c(8000000400000000)
```

**Comment by jandryuk** [comment]
I force pushed a branch - no changes - because Rich Persaud wanted me to use his Ticket numbers.

**Comment by eric-ch** [comment]
Will merge soon.

**Pull request merged**